practice-one/src/pages/Analytics/Analytics.jsx

```diff
@@ -1,7 +1,90 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Analytics.css";
+import { Button, Expand, CustomerItem } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { plusIcon } from "@assets/images";
+import { EXPAND_TITLES } from "@data";
+import { getAllCustomerService } from "../../services/customerService";
+import { v4 as uuidv4 } from "uuid";
 
 const Analytics = () => {
-  return <h1>Analytic</h1>;
+  const [customers, setCustomers] = useState([]);
+  const [error, setError] = useState(null);
+  const [errorPopup, setErrorPopup] = useState(true);
```

Comment:
why need the error popup?

Reply:
I changed to use toast message
![image](https://github.com/datdragon777/React-Training/assets/74806826/d9641089-509c-4b77-9384-57e21b31f43a)